### PR TITLE
Categories

### PR DESCRIPTION
**Pull request by busy-gnl**

### CODE REVIEW

File: public/assets/css/style.css [hunk 1]
```diff
@@ -159,8 +168,29 @@ h5 {
...
+        background: none;
+    }
+
+    .banniere{
+        background: url('/assets/images/vigne.jpg') no-repeat fixed;
+        background-size: 100% auto;
+        height: 50vh;
+        color: #FDF2E9;
+
+    }
```
Comment (aldaitzwild) [inline public/assets/css/style.css:193]: No media queries are allowed, you need to find how to do this with Bootstrap

File: src/Model/WineManager.php [hunk 2]
```diff
@@ -8,16 +8,31 @@ class WineManager extends AbstractManager
 {
     public const TABLE = 'wine';
 
+    public function selectOneWineById(int $id): array|false
+    {
+        // prepared request
+        $statement = $this->pdo->prepare('SELECT wine.id as id, name, year, price, region, color, grape, partner_id, '
+        . static::TABLE . '.description, p.lastname as lastname, p.firstname as firstname FROM ' . static::TABLE .
+        ' INNER JOIN partner as p ON p.id = wine.partner_id WHERE ' . static::TABLE . '.id=:id');
```
Comment (aldaitzwild) [inline src/Model/WineManager.php:16]: instead of putting static::TABLE everywhere, you should put an alias in your query for wine as you do for partner.

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/Model/WineManager.php (hunk 2 above)
Comment (busy-gnl) [reply src/Model/WineManager.php:16]: i admit that's rather ugly

File: public/assets/css/style.css (hunk 1 above)
Comment (busy-gnl) [reply public/assets/css/style.css:193]: >.<"

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**